package.json

```diff
@@ -54,7 +54,8 @@
     "url-search-params": "0.7.1",
     "webpack": "^3.5.2",
     "webpack-dev-server": "^2.7.1",
-    "whatwg-fetch": "^3.0.0"
+    "whatwg-fetch": "^3.0.0",
+    "core-js": "^2.5.7"
```

Comment:
@fgutmann core-js@2.5.7 already ships with `@babel#runtime-corejs2`, there is no need to define it explicitly

Reply:
There is now a transitive dependency to `core-js 3.0.0` from `@babel/preset-env@7.4.2`. The `3.0.0` version seems to have an incompatible directory structure with `2.x`.

```
>$ npm ls core-js
kotlin-playground@1.18.1
├─┬ @babel/preset-env@7.4.2
│ └─┬ core-js-compat@3.0.0
│   └── core-js@3.0.0 
├─┬ @babel/runtime-corejs2@7.4.2
│ └── core-js@2.6.5 
└─┬ babel-plugin-async-to-promises@1.0.5
  └─┬ babel-helper-hoist-variables@6.24.1
    └─┬ babel-runtime@6.26.0
      └── core-js@2.6.5 
```

I'm not sure if adding it explicitly is the best option though. I just wanted to get it building :-)

Reply:
Ah I see. We should downgrade to babel 5.3 then. Could you do it?